server/services/userService.test.ts

```diff
@@ -18,6 +20,9 @@ describe('UserService', () => {
 
   let userService: UserService
 
+  const username = 'JOHN_SMITH'
+  const userResponse = userFactory.build({ name: 'john smith', username })
```

Comment:
I wonder if calling this variable `user` would be more in line with the rest of the codebase?